generators/server/index.js

```diff
@@ -259,7 +259,7 @@ module.exports = class extends BaseGenerator {
 
                     // Generate JWT secret key if key does not already exist in config
                     if (this.authenticationType === 'jwt' && this.jwtSecretKey === undefined) {
-                        this.jwtSecretKey = Buffer.from(crypto.randomBytes(50).toString('hex')).toString('base64');
+                        this.jwtSecretKey = Buffer.from(crypto.randomBytes(100).toString('hex')).toString('base64');
```

Comment:
64 bytes would be sufficient since that's 512 bytes - the max needed for any JWT HMAC SHA algorithm.

Reply:
Thanks!